CMakeCache.txt

```diff
@@ -0,0 +1,95 @@
+# This is the CMakeCache file.
```

Comment:
We don't want to merge any files except *cc, *.h and CMakeList.txt. Other files are generated by CMake and should not be source control.